### ISSUE TITLE
2.6.2 RTMP seekto error

### ISSUE THREAD

**Issue by belivedev** [ksvc/KSYLive_Android#77]
i faced the following error when i try to do several seekto action with rtmp url

`
W/KSYMediaPlayer: SDL_VoutAndroid_releaseBufferProxy_l: [0] ???????? proxy 0: vout: 29 idx: -1 render: false fake: NO
E/com.ksyun.media.player.KSYMediaPlayer: Error (-1004,0)`

please advice.

**Comment by FirmianaRain** [comment]
-1004 means timeout~

**Comment by belivedev** [comment]
yes, my rtmp video is able to play at the beginning. after seekto different position for few times, it shows this error. please advice.
